pos/src/main/java/edu/illinois/cs/cogcomp/pos/lbjava/POSTagger.java

```diff
@@ -75,6 +77,13 @@ public String discreteValue(Object __example) {
         return taggerUnknown.discreteValue(w);
     }
 
+    // TODO(Yewen): find a better way to extract the labels from the classifier.
+    // we tried to extract labels from LabelLexicon, but needed non-trivial amount of post-processing to get the raw
+    // label strings.
+    public Set<String> getTagValues() {
+        return taggerKnown.scores(new Token()).values();
```

Comment:
True. It contains modifications of the actual labels. Like in the case of Saul it contains `LBP_sth(true)` and  `LBP_sth(false)`. Similar issue here in direct usage.